apps/core/src/app.module.ts

```diff
@@ -23,6 +24,7 @@ import { globalConfig } from './config/global.config';
     }),
     DashboardsModule,
     MigrationModule,
+    EdgeLoginModule,
```

Comment:
nit:
the migration and dashboard modules might not be useful without the cognito (JWT) token; I think it warrants a dedicated app module (or dynamic module) for edge mode;
don't need to do it in this PR, we can just log a task for the refactor 🙏